### PR TITLE
CASMPET-6131 add management-nodes-rollout IUF stage

### PR DESCRIPTION
**Pull request by leliasen-hpe**
# Description

add management-nodes-rollout IUF stage.
Initial testing completed on Frigg.

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by rustydb — APPROVED** [review]

**Review by nrockershousen — APPROVED** [review]

**Comment by leliasen-hpe** [comment]
manually backported. Here is 1.4 PR: https://github.com/Cray-HPE/docs-csm/pull/3147

**Review by rustydb — APPROVED** [review]

File: workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml
Comment (cdelatte-hpe) [inline workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml:34]: Do not merge until there is a bigger conversation around this.

**Review by cdelatte-hpe — CHANGES_REQUESTED** [review]

File: workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml
Comment (cdelatte-hpe) [reply workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml:34]: to be held in future conversations.  resolving to unblock.

**Review by cdelatte-hpe — APPROVED** [review]

**Pull request merged**
